e2e-test/src/test/java/io/cdap/plugin/file/locators/CdfFileLocators.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.file.locators;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.How;
+
+/**
+ * File Connector Locators.
+ */
+
+public class CdfFileLocators {
+
+  @FindBy(how = How.XPATH, using = "//*[@placeholder='Name used to identify this source for lineage']")
+  public static WebElement referenceName;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='path' and @class='MuiInputBase-input']")
+  public static WebElement filePath;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='sampleSize' and @class='MuiInputBase-input']")
+  public static WebElement sampleSize;
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Select one')]")
+  public static WebElement format;
+
+  @FindBy(how = How.XPATH, using = "//*[@class='fa fa-remove']")
+  public static WebElement closeButton;
+
+  @FindBy(how = How.XPATH, using = "//*[@title=\"File\"]//following-sibling::div")
+  public static WebElement fileProperties;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy='switch-skipHeader']")
+  public static WebElement skipHeader;
+
+  @FindBy(how = How.XPATH, using = "//*[contains(text(),'Get Schema')]")
+  public static WebElement getSchemaButton;
+
+  @FindBy(how = How.XPATH, using = "//*[@data-cy=\"plugin-File-batchsource\"]")
+  public static WebElement fileBucket;
```

Comment:
'bucket' is associated with object storage. Is this referring to a file path?

Reply:
looks like it is referring to file batch source plugin.

Reply:
fileBucket -> fileBatchSource

Reply:
Renamed.